ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorAliasTokensDemoController.swift

```diff
@@ -0,0 +1,323 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class ColorAliasTokensDemoController: DemoTableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: TableViewCell.identifier)
+        tableView.backgroundColor = .lightGray
```

Comment:
it feels like a bug if the background color in darkmode doesn't change, the title of the democontroller becomes white and immediately it will be against 4.5:1  contrast ratio?

Reply:
Should I make the text stay the same in light/dark or let the background change in dark/light?

Reply:
I understand why you went for the light gray you did, but I think we might be better served by just using the normal table background color.

You mentioned some colors bleed badly with that; can you show an example of where that doesn't work?

Reply:
Sounds good. I have removed the light gray background and updated the description with screenshots.